subsys/bluetooth/common/Kconfig

```diff
@@ -12,8 +12,8 @@ config BT_HAS_HCI_VS
 
 config BT_HCI_VS
 	bool "Zephyr HCI Vendor-Specific Commands"
-	depends on BT_HAS_HCI_VS
-	default y
+	depends on BT_HAS_HCI_VS || !BT_CTLR
```

Comment:
@jhedberg I didn't see your suggestion on the issue before I made this. Should we remove the depends on completely here?